ccd-client/src/main/java/uk/gov/hmcts/cmc/ccd/client/model/CaseDataContent.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.cmc.ccd.client.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
+@Builder
+public class CaseDataContent {
+    private Event event;
+    private JsonNode data;
```

Comment:
could we use object instead of `JsonNode`?
json is an implementation detail for serialisation and shouldn't be a requirement of the model.

same below